apps/src/templates/sectionAssessments/MultipleChoiceAnswerCell.jsx

```diff
@@ -25,14 +28,19 @@ class MultipleChoiceAnswerCell extends Component {
     percentValue: PropTypes.number.isRequired,
     isCorrectAnswer: PropTypes.bool,
     displayAnswer: PropTypes.string,
+    opacity: PropTypes.number,
   };
 
   render() {
-    const {percentValue, isCorrectAnswer, displayAnswer} = this.props;
+    const {percentValue, isCorrectAnswer, displayAnswer, opacity} = this.props;
+
+    const rgbaValue = (isCorrectAnswer) ? {backgroundColor: `rgba(159, 212, 159, ${opacity})`} :
```

Comment:
maybe call this something like `backgroundCSS` since its not just an rgb string?